datahub-web-react/src/app/identity/group/GroupList.tsx

```diff
@@ -17,9 +17,17 @@ import { GROUPS_CREATE_GROUP_ID, GROUPS_INTRO_ID } from '../../onboarding/config
 import { OnboardingTour } from '../../onboarding/OnboardingTour';
 import { addGroupToListGroupsCache, DEFAULT_GROUP_LIST_PAGE_SIZE, removeGroupFromListGroupsCache } from './cacheUtils';
 
-const GroupContainer = styled.div``;
+const GroupContainer = styled.div`
+    display: flex;
+    flex-direction: column;
+    height: 100%;
+    overflow: auto;
```

Comment:
I'm not sure we need this

Reply:
if we remove Flex from GroupContainer scroll will go this upper container. But we can remove height: 100%